content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -24,6 +24,7 @@ All **CronJob** `schedule:` times are based on the timezone of the
 If your control plane runs the kube-controller-manager in Pods or bare
 containers, the timezone set for the kube-controller-manager container determines the timezone
 that the cron job controller uses.
+To specify the CronJob schedule timezone, you can also use [TZ](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
```

Comment:
To specify the timezone of CronJob schedule, you can get the list of timezone from [here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones),  and modify timezone name, such as `UTC`, in CronJob schedule as you need.